packages/key-did-resolver/src/secp256r1.ts

```diff
@@ -0,0 +1,51 @@
+import * as u8a from 'uint8arrays'
+import multicodec from 'multicodec'
+import  multibase from'multibase'
+
+/**
+ * Constructs the document based on the method key
+ */
+export function keyToDidDoc (pubKeyBytes: Uint8Array, fingerprint: string): any {
+  const did = `did:key:${fingerprint}`
+  const keyId = `${did}#${fingerprint}`
+  const key = pubKeyBytesToXY(pubKeyBytes);
+  return {
+    id: did,
+    verificationMethod: [{
+      id: keyId,
+      type: 'JsonWebKey2020',
+      controller: did,
+       publicKeyJwk: {
+         kty: "EC",
+	       crv: "P-256",
+	       x: key.xm,
+	       y: key.ym,
+      }, 
+    }],
+    authentication: [keyId],
+    assertionMethod: [keyId],
+    capabilityDelegation: [keyId],
+    capabilityInvocation: [keyId],
+  }
+  }
+
+function pubKeyBytesToHex(pubKeyBytes: Uint8Array) {
+ const bbf = u8a.toString(pubKeyBytes,'base16')
+ return bbf;
+}
+
+function publicKeyToXY(publicKeyHex: string) {
```

Comment:
This would break if the public key in the did string only contains the x value.

Reply:
I believe that I have the X,Y pair. My did:key is of the same length as the did:key spec.